gravitee-apim-portal-webui/dist-next/browser/polyfills-L5LZC3JG.js

```diff
@@ -0,0 +1,2 @@
a=j(t),l=null;if(r&&(!(l=r[a])||!r.hasOwnProperty(a))){l=r[a]=r[t];let g=r&&pe(r,t);if(Je(g)){let _=o(l,a,t);r[t]=function(){return _(this,arguments)},le(r[t],l)}}return l}function Tt(e,t,o){let r=null;function a(l){let g=l.data;return g.args[g.cbIdx]=function(){l.invoke.apply(this,arguments)},r.apply(g.target,g.args),l}r=ue(e,t,l=>function(g,_){let b=o(g,_);return b.cbIdx>=0&&typeof _[b.cbIdx]=="function"?$e(b.name,_[b.cbIdx],b,a):l.apply(g,_)})}function le(e,t){e[j("OriginalDelegate")]=t}var Ye=!1,De=!1;function Et(){try{let e=Ee.navigator.userAgent;if(e.indexOf("MSIE ")!==-1||e.indexOf("Trident/")!==-1)return!0}catch{}return!1}function gt(){if(Ye)return De;Ye=!0;try{let e=Ee.navigator.userAgent;(e.indexOf("MSIE ")!==-1||e.indexOf("Trident/")!==-1||e.indexOf("Edge/")!==-1)&&(De=!0)}catch{}return De}Zone.__load_patch("ZoneAwarePromise",(e,t,o)=>{let r=Object.getOwnPropertyDescriptor,a=Object.defineProperty;function l(f){if(f&&f.toString===Object.prototype.toString){let u=f.constructor&&f.constructor.name;return(u||"")+": "+JSON.stringify(f)}return f?f.toString():Object.prototype.toString.call(f)}let g=o.symbol,_=[],b=e[g("DISABLE_WRAPPING_UNCAUGHT_PROMISE_REJECTION")]!==!1,k=g("Promise"),m=g("then"),Z="__creationTrace__";o.onUnhandledError=f=>{if(o.showUncaughtError()){let u=f&&f.rejection;u?console.error("Unhandled Promise rejection:",u instanceof Error?u.message:u,"; Zone:",f.zone.name,"; Task:",f.task&&f.task.source,"; Value:",u,u instanceof Error?u.stack:void 0):console.error(f)}},o.microtaskDrainDone=()=>{for(;_.length;){let f=_.shift();try{f.zone.runGuarded(()=>{throw f.throwOriginal?f.rejection:f})}catch(u){C(u)}}};let O=g("unhandledPromiseRejectionHandler");function C(f){o.onUnhandledError(f);try{let u=t[O];typeof u=="function"&&u.call(this,f)}catch{}}function G(f){return f&&f.then}function A(f){return f}function K(f){return n.reject(f)}let F=g("state"),R=g("value"),T=g("finally"),Y=g("parentPromiseValue"),x=g("parentPromiseState"),W="Promise.then",M=null,E=!0,$=!1,d=0;function D(f,u){return c=>{try{q(f,u,c)}catch(h){q(f,!1,h)}}}let w=function(){let f=!1;return function(c){return function(){f||(f=!0,c.apply(null,arguments))}}},J="Promise resolved with itself",oe=g("currentTaskTrace");function q(f,u,c){let h=w();if(f===c)throw new TypeError(J);if(f[F]===M){let y=null;try{(typeof c=="object"||typeof c=="function")&&(y=c&&c.then)}catch(P){return h(()=>{q(f,!1,P)})(),f}if(u!==$&&c instanceof n&&c.hasOwnProperty(F)&&c.hasOwnProperty(R)&&c[F]!==M)ne(c),q(f,c[F],c[R]);else if(u!==$&&typeof y=="function")try{y.call(c,h(D(f,u)),h(D(f,!1)))}catch(P){h(()=>{q(f,!1,P)})()}else{f[F]=u;let P=f[R];if(f[R]=c,f[T]===T&&u===E&&(f[F]=f[x],f[R]=f[Y]),u===$&&c instanceof Error){let p=t.currentTask&&t.currentTask.data&&t.currentTask.data[Z];p&&a(c,oe,{configurable:!0,enumerable:!1,writable:!0,value:p})}for(let p=0;p<P.length;)ee(f,P[p++],P[p++],P[p++],P[p++]);if(P.length==0&&u==$){f[F]=d;let p=c;try{throw new Error("Uncaught (in promise): "+l(c)+(c&&c.stack?`
```

Comment:
### :warning: Aqua detected sensitive data in your code  
**Category:** Azure   
**Description:** Azure SAS Token   
**Severity:** HIGH   
**Match:** y loaded.");return e.Zone}let ******** se{static assertZon  
[This comment was created by Aqua Pipeline]